scripts/cdn/branch/unpublish.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Unpublish packages from our CDN
+ */
+
+// Load secrets file
+
+let fs = require('fs');
+
+let secrets = JSON.parse(fs.readFileSync('secrets.json'));
+if (!secrets ||
+    !secrets.AWS_USER_S3_KEY ||
+    !secrets.AWS_USER_S3_SECRET ||
+    !secrets.AWS_S3_BUCKET) {
+    throw Error('secrets.json content is not valid');
+}
+
+const branchName = require('current-git-branch');
+const branch = JSON.parse(fs.readFileSync('package.json')).branch || branchName();
+
+let s3 = require('s3');
+let client = s3.createClient({
+    s3Options: {
+        accessKeyId: secrets.AWS_USER_S3_KEY,
+        secretAccessKey: secrets.AWS_USER_S3_SECRET
+    }
+});
+
+deleteFiles(branch);
+
+function deleteFiles (branch) {
+    console.log('Unpublishing branch:', branch);
+
+    let uploader = client.deleteDir({
```

Comment:
I'm super scared by this

Reply:
Why? We can add a condition checking the `branch` parameter is not a version. What do you think?

Reply:
I would just leave the published branches in S3. If we don't want to put so much noise there, we could use a subfolder in which we put all the branches (but not the real versions)

Reply:
In my opinion it doesn't add any noise and it's very useful to keep things clean. If we don't want to have this because it's dangerous, for me it's ok, but if it's because it adds 'noise' I don't agree.